### PR TITLE
UN-2212 Command Template for Removing Extra BitLocker Numerical Passwords

### PR DESCRIPTION
**Pull request by szanjanizadehjc**
## Issues
* [UN-2212](https://jumpcloud.atlassian.net/browse/UN-2212) - Command Template for Removing Extra BitLocker Numerical Passwords

## What does this solve?
Adds a Powershell script to remove extra Bitlocker recovery passwords.

## Is there anything particularly tricky?
This PR should go after [UN-2288](https://jumpcloud.atlassian.net/browse/UN-2288).

## How should this be tested?
Bitlocker policy should be activated on a windows device.  After the OS drive is encrypted, in an Administrator Powershell command window type the following to create an extra Recovery Password:
`manage-bde -protectors -add -rp c:`
Then type the following to make sure there are two Numerical Passwords:
`manage-bde -protectors -get c:`
Run the command template from the Admin Portal to remove the extra password.  Run the previous command in a Powershell window to make sure there is only one Numerical Password.

## Screenshots

[UN-2212]: https://jumpcloud.atlassian.net/browse/UN-2212?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

[UN-2288]: https://jumpcloud.atlassian.net/browse/UN-2288?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: PowerShell/JumpCloud Commands Gallery/Windows Commands/Windows - Remove Extra Bitlocker Recovery Passwords.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+function Remove-NonMatchingBitLockerRecoveryPasswords {
+  [CmdletBinding()]
+  param(
+    [Parameter(Mandatory=$true)]
+    [ValidateNotNull()]
+    [string] $RecoveryKey
+  )
+
+  try {
+	$osVolumeLetter = Get-OsVolumeLetter
```
Comment (weastonjc) [inline PowerShell/JumpCloud Commands Gallery/Windows Commands/Windows - Remove Extra Bitlocker Recovery Passwords.md:28]: tabbing seems to be off in when displayed in the rich diff.  Not a crazy problem, but could make the code a bit hard to read through

**Review by weastonjc — CHANGES_REQUESTED** [review]
see comment

File: PowerShell/JumpCloud Commands Gallery/Windows Commands/Windows - Remove Extra Bitlocker Recovery Passwords.md (hunk 1 above)
Comment (szanjanizadehjc) [reply PowerShell/JumpCloud Commands Gallery/Windows Commands/Windows - Remove Extra Bitlocker Recovery Passwords.md:28]: Good catch.

**Review by weastonjc — DISMISSED** [review]

**Review by gweinjc — DISMISSED** [review]
Appears to be working on my end. Nice job

**Review by kmaranionjc — DISMISSED** [review]
New change tested and working on my test env. Great work!

**Review by weastonjc — APPROVED** [review]

**Review by nwannerjc — APPROVED** [review]

**Review by gweinjc — APPROVED** [review]

**Review by kmaranionjc — APPROVED** [review]

**Comment by jworkmanjc** [comment]
@szanjanizadehjc feel free to merge this in, looks like you have everything you need.

**Comment by szanjanizadehjc** [comment]
Unfortunately, this change is blocked by un-2288.  I can’t have this change in production and customers using it until un-2288 is in production.


Sahab Zanjanizadeh
Software Engineer 3
***@***.*** ***@***.***>
859.391.0048
___________________________________
   <https://www.linkedin.com/company/jumpcloud>
Watch this short video <https://www.youtube.com/watch?v=SL9lrLv8Vcc> to learn how a Zero Trust Security model maintains secure access in a domainless world.

The information transmitted is intended only for the person to whom it is addressed and may contain confidential material. Review or other use of this information by persons other than the intended recipient is prohibited. If you've received this in error, please contact the sender and delete it from any computer.

**Pull request merged**